.github/workflows/deploy.yml

```
@@ -0,0 +1,43 @@
+name: Deploy to AWS
+
+on:
+  push:
+    branches:
+      - dev
+      - main
+  workflow_dispatch:
+
+jobs:
+  deploy_to_aws:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Check out the application's source code
+        uses: actions/checkout@v2
+
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Install application dependencies
+        run: npm ci
+
+      - name: Deploy - dev
+        if: github.ref == 'refs/heads/dev'
+        run: npm run deploy
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.DEV_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.DEV_AWS_SECRET_ACCESS_KEY }}
+          AWS_ACCOUNT_ID: ${{ secrets.DEV_AWS_ACCOUNT_ID }}
+          DEPLOY_ENVIRONMENT: dev
+
+      - name: Deploy - production
+        if: github.ref == 'refs/heads/main'
+        run: |
+          npm run deploy
```

This pipe might be unnecessary?

```suggestion
        run: npm run deploy
```

---

It is, it was added when running a test task and setting debug mode before deploying.